ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/components/ScanConfigClustersTable.tsx

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable react/jsx-no-comment-textnodes */
+// eslint-disable @typescript-eslint/ban-ts-comment
+import React, { useState } from 'react';
+import {
+    Card,
+    CardActions,
+    CardBody,
+    CardHeader,
+    CardTitle,
+    Pagination,
+} from '@patternfly/react-core';
+import { TableComposable, Tbody, Td, Th, ThProps, Thead, Tr } from '@patternfly/react-table';
+
+import IconText from 'Components/PatternFly/IconText/IconText';
+import { ClusterScanStatus } from 'services/ComplianceEnhancedService';
+import { getClusterStatusObject } from '../compliance.scanConfigs.utils';
+
+type ScanConfigClustersTableProps = {
+    clusterScanStatuses: ClusterScanStatus[];
+};
+
+function ScanConfigClustersTable({ clusterScanStatuses }: ScanConfigClustersTableProps) {
+    const [page, setPage] = useState(1);
+    const [perPage, setPerPage] = useState(20);
+
+    const onSetPage = (
+        _event: React.MouseEvent | React.KeyboardEvent | MouseEvent,
+        newPage: number
+    ) => {
+        setPage(newPage);
+    };
+
+    const onPerPageSelect = (
+        _event: React.MouseEvent | React.KeyboardEvent | MouseEvent,
+        newPerPage: number,
+        newPage: number
+    ) => {
+        setPerPage(newPerPage);
+        setPage(newPage);
+    };
+
+    // Index of the currently sorted column
+    // Note: if you intend to make columns reorderable, you may instead want to use a non-numeric key
+    // as the identifier of the sorted column. See the "Compound expandable" example.
+    const [activeSortIndex, setActiveSortIndex] = useState<number>(0);
+
+    // Sort direction of the currently sorted column
+    const [activeSortDirection, setActiveSortDirection] = useState<'asc' | 'desc'>('asc');
+
+    // Since OnSort specifies sorted columns by index, we need sortable values for our object by column index.
+    // This example is trivial since our data objects just contain strings, but if the data was more complex
+    // this would be a place to return simplified string or number versions of each column to sort by.
+    const getSortableRowValues = (cluster: ClusterScanStatus): (string | number | null)[] => {
+        // @ts-expect-error api
+        const { clusterName, provider, status } = cluster;
+
+        return [clusterName, null, provider, status] as (string | number | null)[];
```

Comment:
Too bad there isn't an eslint equivalent to `@ts-expect-error` that would error out here once the API is implemented, as an alternative to the `as` type assertion.